GroupProject/PlayerInfoSchema.json

```diff
@@ -21,10 +21,14 @@
           "type": "integer"
         },
         "PlayerName": {
-          "type": "string"
+          "type": "string",
+          "minLength": 4,
+          "maxLength":  45
         },
         "TeamName": {
-          "$ref": "#/definitions/TeamName"
+          "$ref": "#/definitions/TeamName",
```

Comment:
Added length validation for TeamName. Although validation does not seem to be integrated into Index page yet, this will validate the string has a length between the values below.

Source: https://json-schema.org/understanding-json-schema/reference/string.html